doc/release-notes/release-4.sgml

```diff
@@ -176,10 +176,14 @@ Most user-facing changes are reflected in squid.conf (see below).
 
 
 <sect1>Initial GnuTLS support
-<p>If all you need is a proxy that connects over TLS/SSL to a <em>cache_peer</em>
-   or accepts <em>https://</em> URLs over clear-text and performs the necessary
-   upstream TLS connections. Then you now have the choice to build Squid with
-   GnuTLS instead of OpenSSL.
+<p>Squid can now be built to use GnuTLS in place of OpenSSL for the core
+   features of receiving TLS connections from clients, and making TLS
+   connection to servers. The GnuTLS support is still very much experimental
+   and should be tested before use.
+
+<p>Also, be aware that the feature implementation is far from complete. The
```

Comment:
You can drop the `Also, be aware that` fluff.

It is not clear what `the feature` is referring to. The earlier text talked about accepting and establishing TLS connections. It is not clear whether _that_ implementation is incomplete, whether the SslBump features mentioned after this sentence are the missing parts, or whether some other important pieces are missing. What you have probably meant can be expressed like this:

> GnuTLS coverage (of Squid features) is far from complete: SslBump and many other TLS-related features are not yet supported by GnuTLS builds.

You can keep or drop the parenthesized portion, depending on how precise you want this statement to be.